src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -542,6 +542,73 @@ bool InputProcessor::getDefaultValue(EnergyPlusData &state, std::string const &o
     return defaultFound;
 }
 
+std::string InputProcessor::getAlphaFieldValue(json const &ep_object, json const &schema_obj_props, std::string const &fieldName)
+{
+    auto const &schema_field_obj = schema_obj_props[fieldName];
+    assert(!schema_field_obj.empty()); // Check that field name exists in the schema for this object type
```

Comment:
Why did you convert this to an `assert` statement instead of the `FatalErrorMessage`? Asserts only work in debug builds, so is this something only developers will ever hit? If not, then it should probably be the FatalError.

Reply:
I changed to `assert` because this is a developer error (same for the others). The function arguments for `state` and `objectType` were only being used for these error messages that should never happen unless the developer makes a mistake. So it seemed reasonable to use an `assert` and drop those.

Reply:
@mjwitte can you give me an example of why schema_field_obj.empty() would ever be false? I get the misspelled field name issue but are there other instances? Inputs past min-fields?

Reply:
I see this is merged, but just for the record, this line is looking at the schema (not the user input) in `schema_obj_props`. So `empty` will be true if the field name does not exist in `schema_obj_props`.